src/main/java/Sweh.java

```diff
@@ -9,8 +23,15 @@
 import static util.Parser.getArgMap;
 import static util.Parser.getCommand;
 
-public class Sweh {
+public class Sweh extends Application {
     public static TaskList taskList = new TaskList();
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
```

Comment:
I like that you use "user" and "duke" to differentiate the respective images. However, would it be more appropriate to name them as "userImage" and "dukeImage"? As "user" and "duke" might be a more suitable name for the respective Object Class.